Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [workspace]
-resolver = "1"
+resolver = "2"
```

Comment:
Just FYI, I don't think this change is feasible without doing a lot of work.

Reply:
Yeah, the CI run seems to think the same :)